### DISCUSSION TITLE
Quality search/filter appears to work as "greater than" and not "equals"

### DISCUSSION THREAD

**Discussion by andrewdolphin** [photoprism/photoprism#3639] [Q&A]
When using the search bar to filter/search by image quality (e.g. quality:2 as described in the documentation) it appears to pick up all images of quality two or greater.

This makes it impossible to filter out poor quality images.

If it operated as an equals you could combine filters (e.g. quality:0 and quality:1)

**Comment by lastzero** [comment]
If the quality is 2 or less, you can simply use our review filter and functionality.

**Comment by lastzero** [comment]
Note 0 is the default (unknown) and so won't be something you'll see in practice. -1 means the picture is broken and you can find it in Library > Hidden.

**Comment by graciousgrey** [comment]
We can improve the documentation that it is more clear that this filter means greater than or equal and not equal
